spec/controllers/followings_controller_spec.rb

```diff
@@ -0,0 +1,80 @@
+require 'spec_helper'
+
+describe FollowingsController do
+  before(:each) do
+    request.env["HTTP_REFERER"] = "previous_page"
+  end
+  describe "GET people" do
+    context "with authenticated users" do
+      let(:following) { Fabricate(:following, follower_id: current_user.id) }
```

Comment:
When you only have a single spec, using `let` and `before` seems like more work than just putting the code in the spec. While `let` and `before` make your code nicer to look at, they also introduce abstraction that can make your code harder to follow. I speak from experience, I am currently dealing with overuse of both in my own app.